gbpservice/neutron/nsf/modules/config_agent.py

```diff
@@ -0,0 +1,75 @@
+import os
```

Comment:
This file should be under nsf/config_agent/modules. This should be consistent with the dir structure for lifecycle_manager and configurator.